data/dfndata/npc/undead.dfn

```diff
@@ -479,3 +492,31 @@ NAME=#//a ghoul
 ID=0x0099
 SKIN=0x0
 }
+
+//lady of the snow - Base NPC with shared DFN tags
+[base_lady_of_the_snow]
+{
+GET=base_undead
+NAME=#//lady of the snow
+ID=0x0fc
+ORIGIN=se
+}
+
+[ladyofthesnow]
+{
+GETSE=ladyofthesnow_se
+}
+
+//revenant lion - Base NPC with shared DFN tags
+[base_revenant_lion]
+{
+GET=base_undead
+NAME=#//revenant lion
+ID=0x0fb
+ORIGIN=se
+}
+
+[revenantlion]
+{
+GETSE=revenant_lion_se
```

Comment:
Add one of these for each expansion, so adding NPC with wrong expansion selected doesn't break the server. We can update where these point at when we have definitions for different eras:

GETUO=revenant_lion_se
GETT2A=revenant_lion_se
GETUOR=revenant_lion_se
GETTD=revenant_lion_se
GETLBR=revenant_lion_se
GETAOS=revenant_lion_se
GETSE=revenant_lion_se
GETML=revenant_lion_se
GETSA=revenant_lion_se
GETHS=revenant_lion_se
GETTOL=revenant_lion_se